### DISCUSSION TITLE
doris版本V1.0 使用debezium实时同步insert into values方式，BE节点CPU一直超过200%以上

### DISCUSSION THREAD

**Discussion by fuckjavaer** [apache/doris#19703] [RFC]
doris版本V1.0 使用debezium实时同步insert into values方式，BE节点CPU一直超过200%以上，请教各位大佬，有什么解决方案，实时同步的问题

**Comment by LemonLiTree** [comment]
1. 不要使用insert into values的方式写入doris，compaction压力会很大，导致性能下降甚至oom的问题。使用stream load的方式写
2. 推荐使用flink cdc 写，doris connector都封装好了。
3. 推荐升级，1.1 版本有很多compaction的优化
